### ISSUE TITLE
Changing date format doesn't have effect

### ISSUE THREAD

**Issue by jarss** [Zren/plasma-applet-eventcalendar#81]
Doesn't seem like any of the changes I apply in the settings (under "general") take effect. Date format stays the same. 

Would really want a dd-MM-yyyy format (30-05-209). And, while at it if at all possible, week number (don't mind going smaller fonts, but changing font also has no affect). 

`[Screenshot](https://i.imgur.com/TF6dreV.png)`

**Comment by Zren** [comment]
What locale do you use? en_GB?

I might end up changing the default to europe order, then overriding then reversing it in the en_US and en_CA locales.

That or I should dynamically reorder them based on if `(dd)` comes before `(MM)` in the short date format.

**Comment by jarss** [comment]
I'm not sure what you refer to, sorry. 
When I'm looking for Locale I come up with "Locale - System Setting Module," which I set to English (United States) en_US.UTF-8. Would that be it? 

I have the option to change to en_GB. which I did just to try, that doesn't' seem to do anything. I need to sign off and in again.

**Comment by jarss** [comment]
I'm not sure what you refer to, sorry. 
When I'm looking for Locale I come up with "Locale - System Setting Module," which I set to English (United States) en_US.UTF-8. Would that be it? aa

I have the option to change to en_GB. which I did just to try, that doesn't' seem to do anything. I signed in and out.

**Comment by flying-sheep** [comment]
Yeah, it doesn’t seem to use the system setting for the calendar date. I also see an American date format, which looks weird to a German: „Juli 20, 2020“ is simply not correct, it should be „20. Juli 2020“.
